docs/REST API/rest-api.md

```diff
@@ -9901,8 +9870,136 @@ Only one of `labels_include_any` or `labels_exclude_any` can be specified. If ne
 }
 ```
 
+### Patch software
```

Comment:
@noahtalerman I'd reconsider the naming of this one, just because "patch" is kind of a loaded term when looking @ API docs. What this really does is modify a software title by setting the value of `enable_patch`, so it feels strange to me to have this be a `POST` endpoint. Is there some kind of "Modify ____" endpoint we could have this be instead? Happy to jump on the next #g-software design review to discuss.